modules/grpc/otel/Makefile.am

```diff
@@ -1,18 +1,60 @@
+OTEL_PROTO_SRCDIR = $(top_srcdir)/modules/grpc/otel/opentelemetry-proto
+OTEL_PROTO_BUILDDIR = $(top_builddir)/modules/grpc/otel/proto
+
 if ENABLE_GRPC
 
+OTEL_PROTO_PROTOC_FLAGS = $(PROTOC_GEN_GRPC_CPP_PLUGIN_FLAGS) --proto_path=$(OTEL_PROTO_SRCDIR) --cpp_out=$(OTEL_PROTO_BUILDDIR)
+
+modules/grpc/otel/proto/%.grpc.pb.cc modules/grpc/otel/proto/%.grpc.pb.h:
+	mkdir -p $(OTEL_PROTO_BUILDDIR)
+	$(PROTOC) $(OTEL_PROTO_PROTOC_FLAGS) --grpc-cpp_out=$(OTEL_PROTO_BUILDDIR) $*.proto
+
+modules/grpc/otel/proto/%.pb.cc modules/grpc/otel/proto/%.pb.h:
+	mkdir -p $(OTEL_PROTO_BUILDDIR)
+	$(PROTOC) $(OTEL_PROTO_PROTOC_FLAGS) $*.proto
+
+OTEL_PROTO_BUILT_SOURCES = \
+  $(OTEL_PROTO_BUILDDIR)/opentelemetry/proto/common/v1/common.pb.h \
+  $(OTEL_PROTO_BUILDDIR)/opentelemetry/proto/common/v1/common.pb.cc \
+  $(OTEL_PROTO_BUILDDIR)/opentelemetry/proto/resource/v1/resource.pb.h \
+  $(OTEL_PROTO_BUILDDIR)/opentelemetry/proto/resource/v1/resource.pb.cc \
+  $(OTEL_PROTO_BUILDDIR)/opentelemetry/proto/logs/v1/logs.pb.h \
+  $(OTEL_PROTO_BUILDDIR)/opentelemetry/proto/logs/v1/logs.pb.cc \
+  $(OTEL_PROTO_BUILDDIR)/opentelemetry/proto/metrics/v1/metrics.pb.h \
+  $(OTEL_PROTO_BUILDDIR)/opentelemetry/proto/metrics/v1/metrics.pb.cc \
+  $(OTEL_PROTO_BUILDDIR)/opentelemetry/proto/trace/v1/trace.pb.h \
+  $(OTEL_PROTO_BUILDDIR)/opentelemetry/proto/trace/v1/trace.pb.cc \
+  $(OTEL_PROTO_BUILDDIR)/opentelemetry/proto/collector/logs/v1/logs_service.grpc.pb.h \
+  $(OTEL_PROTO_BUILDDIR)/opentelemetry/proto/collector/logs/v1/logs_service.grpc.pb.cc \
+  $(OTEL_PROTO_BUILDDIR)/opentelemetry/proto/collector/logs/v1/logs_service.pb.h \
+  $(OTEL_PROTO_BUILDDIR)/opentelemetry/proto/collector/logs/v1/logs_service.pb.cc \
+  $(OTEL_PROTO_BUILDDIR)/opentelemetry/proto/collector/metrics/v1/metrics_service.grpc.pb.h \
+  $(OTEL_PROTO_BUILDDIR)/opentelemetry/proto/collector/metrics/v1/metrics_service.grpc.pb.cc \
+  $(OTEL_PROTO_BUILDDIR)/opentelemetry/proto/collector/metrics/v1/metrics_service.pb.h \
+  $(OTEL_PROTO_BUILDDIR)/opentelemetry/proto/collector/metrics/v1/metrics_service.pb.cc \
+  $(OTEL_PROTO_BUILDDIR)/opentelemetry/proto/collector/trace/v1/trace_service.grpc.pb.h \
+  $(OTEL_PROTO_BUILDDIR)/opentelemetry/proto/collector/trace/v1/trace_service.grpc.pb.cc \
+  $(OTEL_PROTO_BUILDDIR)/opentelemetry/proto/collector/trace/v1/trace_service.pb.h \
+  $(OTEL_PROTO_BUILDDIR)/opentelemetry/proto/collector/trace/v1/trace_service.pb.cc
```

Comment:
Can we apply this patch to your PR?

https://github.com/MrAnno/syslog-ng/commit/f2fa334f4b9383334f6b86df3d5082869565fc87

Reply:
Yes, applied.

Reply:
Thanks.